spec/facility_spec.rb

```diff
@@ -2,25 +2,276 @@
   end
   describe '#initialize' do
     it 'can initialize' do
-      expect(@facility).to be_an_instance_of(Facility)
-      expect(@facility.name).to eq('DMV Tremont Branch')
-      expect(@facility.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
-      expect(@facility.phone).to eq('(720) 865-4600')
-      expect(@facility.services).to eq([])
+      expect(@facility_1).to be_an_instance_of(Facility)
+      expect(@facility_1.name).to eq('DMV Tremont Branch')
+      expect(@facility_1.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+      expect(@facility_1.phone).to eq('(720) 865-4600')
+      expect(@facility_1.services).to eq([])
     end
   end
 
   describe '#add service' do
     it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Renew Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    end
+  end
+
+  describe '#register a vehicle' do
+    it 'can register vehicles' do
+      expect(@facility_1.services).to eq []
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to include 'Vehicle Registration'
+    end
+
+    it 'adds registration info for the vehicle' do
+      @facility_1.add_service('Vehicle Registration')
+      expect(@cruz.registration_date).to be nil
+      
+      @facility_1.register_vehicle(@cruz)
+      expect(@cruz.registration_date).to be_an_instance_of Time
+    end
+
+    it 'adds vehicle to registered vehicles list' do
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.registered_vehicles).to eq []
+
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.registered_vehicles).to include(Vehicle)
+      expect(@facility_1.registered_vehicles.count).to eq 1
+
+      @facility_1.register_vehicle(@bolt)
+      @facility_1.register_vehicle(@camaro)
+      expect(@facility_1.registered_vehicles.count).to eq 3
+      
+    end
+
+    it 'collects fees based off vehicle type and age' do
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.collected_fees).to eq 0
+
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.collected_fees).to eq 100
+
+      @facility_1.register_vehicle(@bolt)
+      expect(@facility_1.collected_fees).to eq 300
+
+      @facility_1.register_vehicle(@camaro)
+      expect(@facility_1.collected_fees).to eq 325
```

Comment:
Nice job testing that not only can we calculate one registered vehicle's fees, but it successfully adds those fees together.